src/main.rs

```diff
@@ -91,3 +118,14 @@ async fn setup(
 		pg: pg_client,
 	})
 }
+
+async fn handle_shutdown(shard_manager: Arc<serenity::ShardManager>, reason: ShutdownReason) {
+	let reason = match reason {
+		ShutdownReason::CtrlC => "Interrupted",
+		ShutdownReason::Sigterm => "Received SIGTERM",
+	};
+
+	warn!("{reason}! Shutting down bot...");
+	shard_manager.shutdown_all().await;
+	println!("Everything is shutdown. Goodbye!");
```

Comment:
why not `info!`?

Reply:
It's just to bring attention to it

Reply:
Best show this even if info messages are hidden